hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -680,7 +680,7 @@
       uvm_test_seq: chip_sw_base_vseq
       sw_images: ["//sw/device/tests:otbn_randomness_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+sw_test_timeout_ns=18_000_000"]
+      run_opts: ["+sw_test_timeout_ns=18_000_000","+rng_srate_value=30"]
```

Comment:
Do you want to wait for the PR that adds the `rng_srate_value_min|max` window for randomization?

Reply:
ah good point. I don't mind either way. If the rng stuff lands later I can come and modify all of this.